packages/exchange-consumer/src/transformers/DMMTransformer.ts

```diff
@@ -192,6 +192,7 @@ export class DMMTransformer {
       vcs: vcs
         ? DMMTransformer.grpcVCSResultRecordToVCSResultRecords(vcs)
         : undefined,
+      createdAt: '', // epochResultRecord.getCreatedAt(),
```

Comment:
Did some hack here to ensure the build pass, will update once the `@injectivelabs/exchange-api` library is updated.